styles/styles.css

```diff
@@ -25,6 +25,7 @@
   --primary-blue-100: #012740;
   --primary-blue-101: #002035;
   --primary-blue-102: #001C2D;
+  --primary-blue-103: #002337;
```

Comment:
If this is not part of their design system library, I would suggest naming it entirely differently to avoid confusions.